string name
string frame_id
float32 x
float32 y
float32 z
float32 length
float32 width
float32 height
float32 roll
float32 pitch
float32 yaw
---
std_msgs/Empty empty